src/js/rx/components/Prescription.jsx

```diff
@@ -15,35 +15,40 @@ class Prescription extends React.Component {
     const name = attrs['prescription-name'];
     const remaining = attrs['refill-remaining'];
 
-    let action;
-    let messageProvider;
+    let action = [];
 
-    // TODO: Refillable is currently always false.
-    // Switch to using refillable when it's working.
-    if (remaining === 0) {
-      messageProvider = <MessageProviderLink/>;
+    if (attrs['is-refillable'] === true) {
+      action.push(<SubmitButton
+          cssClass="usa-button-outline rx-prescription-button"
+          text="Refill Prescription"/>);
     } else {
-      action = (
-        <SubmitButton
-            cssClass="usa-button-outline rx-prescription-button"
-            text="Refill Prescription"/>
-      );
-    }
+      if (attrs['refill-status'] !== 'active') {
+        action.push(<div className="rx-prescription-status">{rxStatuses[attrs['refill-status']]}</div>);
 
-    if (attrs['is-trackable']) {
-      action = (
-        <TrackPackageLink
-            className="usa-button"
-            text="Track package"/>
-      );
-    } else {
-      action = <div className="rx-prescription-refill-requested">Refill requested</div>;
+        if (attrs['refill-status'] !== 'submitted') {
+          action.push(<div>Call Provider</div>);
+        }
+      } else {
+        if (attrs['is-trackable'] === true) {
+          action.push(<TrackPackageLink
+              className="usa-button"
+              text="Track package"/>);
+        } else {
+          action.push(<div
+              className="rx-prescription-refill-requested">Refill
+          requested</div>);
+        }
+
+        if (remaining === 0) {
```

Comment:
That should probably say "Call Provider" instead, but yes, it's possible to both [track an order](https://github.com/department-of-veterans-affairs/kudos-team/blob/master/Prescriptions/design/Rx%20statuses.md) and request a prescription renewal.